app/scripts/controllers/users.js

```diff
@@ -8,47 +8,76 @@
  * Controller for the login page
  */
 angular.module('flightNodeDemo')
-	.controller('UserController', function ($scope, $http, $log) {
+	.controller('UserController', ['$scope', '$http', function ($scope, $http, $log) {
 
 		$scope.loading = true;
 
-		$scope.userList = {};
+		$scope.userList = [];
 
-		var jwt = localStorage.getItem('jwt');
+		var jwt = JSON.parse(localStorage.getItem('jwt'));
 
 		if (jwt) {
 			// TODO: validate that the user administration claim is present.
 			// TODO: validate the expiration date.
-			var access_token = (JSON.parse(jwt)).data.access_token;
-			
-			$http({
-				url: 'http://localhost:50323/api/v1/user',
-				method: 'GET',
-				headers: {
-					Authorization: 'bearer ' + access_token
-				}
-			})
-				.then(function success(response) {
-
-					$scope.userList.data = response.data;
-
-				}, function error(response) {
-
-					$log.error(response);
-
-					if (response.status === 401) {
-						$scope.unauthorized();
-					} else {
-						$scope.showErrorMessage({ error: response });
-					}
-				});
+			if (moment() < moment(jwt.expiresAt)) {
 
+				$http({
+					url: 'http://localhost:50323/api/v1/user',
+					method: 'GET',
+					headers: {
+						Authorization: 'bearer ' + jwt.access_token
+					}
+				})
+					.then(function success(response) {
+
+						$scope.userList = _.map(response.data, function (user) {
+							return {
+								fullName: user.givenName + ' ' + user.familyName,
+								email: user.email,
+								phone: (user.mobilePhoneNumber || user.phoneNumber),
+								userId: user.userId
```

Comment:
Reformatting the data, so that we load only a few fields - including the concatenated `fullName`.